css/at-rules/property.json

```
@@ -0,0 +1,55 @@
+{
+  "css": {
+    "at-rules": {
+      "property": {
+        "__compat": {
+          "description": "<code>@property</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@property",
+          "support": {
+            "chrome": {
+              "version_added": "78"
+            },
+            "chrome_android": {
+              "version_added": "78"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```

Since we're worrying about Opera data in the other file, perhaps we should add the Opera version here too?
```suggestion
              "version_added": "65"
```